docs/content/reference/metadata.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "MMv1 metadata reference"
+weight: 35
+---
+
+# MMv1 metadata reference
+
+This page documents all properties for metadata. Metadata does not impact the provider itself, but is used by Google internally for coverage metrics.
```

Comment:
Drive-by suggestion: maybe state this more generically (because that might not be the only thing it gets used for)

```suggestion
This page documents all properties for metadata. Metadata does not impact the provider itself, but enables analysis of the provider that would otherwise be impractical or impossible.
```